src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -282,6 +282,45 @@ describe('successful non-empty recommendations list table', () => {
       .find('td[data-label=Category]')
       .should('contain', 'Service Availability');
   });
+
+  it('the Impacted filters work correctly', () => {
+    const RECS_LIST_TABLE = 'div[id=recs-list-table]';
+    const FILTERS_DROPDOWN = 'ul[class=pf-c-dropdown__menu]';
+    const FILTER_TOGGLE = 'span[class=pf-c-select__toggle-arrow]';
+
+    cy.get(RECS_LIST_TABLE).find('button[class=pf-c-dropdown__toggle]').click();
+    cy.get(FILTERS_DROPDOWN)
+      .contains('Clusters impacted')
+      .then((element) => {
+        cy.wrap(element);
+        element[0].click();
+      });
+    cy.get(FILTER_TOGGLE).then((element) => {
+      cy.wrap(element);
+      element[0].click();
+    });
+    cy.get('#pf-random-id-72-false').check({ force: true });
+    cy.get('.pf-c-chip-group__list-item').contains('1 or more');
+
+    cy.get(RECS_LIST_TABLE).find('button[class=pf-c-dropdown__toggle]').click();
+    cy.get(FILTERS_DROPDOWN)
+      .contains('Status')
+      .then((element) => {
+        cy.wrap(element);
+        element[0].click();
+      });
+    cy.get(FILTER_TOGGLE).then((element) => {
+      cy.wrap(element);
+      element[0].click();
+    });
+    cy.get('button[class=pf-c-select__menu-item]')
+      .contains('All')
+      .then((element) => {
+        cy.wrap(element);
+        element[0].click();
+      });
+    cy.get('.pf-c-chip-group__list-item').contains('1 or more');
```

Comment:
Maybe better check whether both filter groups are present instead of checking just one (Impacting)? In other words, at the end of the scenario, you test whether no groups/filters are gone.